### ISSUE TITLE
Feature: portable aliases (e.g. singing calls as patter, without copying files)

### ISSUE THREAD

**Issue by mpogue2** [mpogue2/SquareDesk#125]
As a general principle, I would like to keep the Music Directory 100% portable between Platforms.  I have mine cross-mounted from Win10 running in Parallels, and it works perfectly.  We can also copy it anywhere right now, and it will work.  

This also means that we can just copy an entire Music Directory onto a flash drive, and use it in Guest Mode.  But, since flash drives are FAT32, they do not allow either symbolic or hard links.  I am inclined to NOT support aliases, because of this.

If we really want to be able to alias singing calls to patter, we'll need to create our own alias concept, one that is portable between all OS's, rather than using underlying OS support for symbolic and/or hard links.

So, this feature request is for some sort of "alias in the database"....If I have a singing call, and I right click "Make alias in...", maybe a dialog pops up and asks me which Type to make it.  It then appears just like any file, although no separate file exists.  The alias would have its own Tempo, Pitch, etc, but wouldn't exist on disk.  There would need to be a "Remove Alias" function, under the right click menu, too.

The short term workaround (although it does take extra space) is to duplicate the MP3 file, and stick it where you want it.  The proposal above is to save space over that scheme.

**Comment by phoef** [comment]
MacOS 10.13.3, SquareDesk V0.9.2alpha6
Made an alias of a Singing file, cut&paste it to the patter directory. 
My observations:
It is displayed twice. One as singing, one as patter. 
If I play the patter it doesn't set the loop and it searched and filled the lyrics tab.

**Comment by mpogue2** [comment]
Yep -- using the Mac-OS-specific "alias" has problems.  Not only is it not portable, but it also doesn't work correctly...Thanks for the report Peter!  I'm adding a Bug tag to this one, too.

**Comment by phoef** [comment]
It seems to be exactly the same when you copy the file. Perhaps not the alias but the existence of the same filename as singing call is the cause? Mmmh, but I also have tests, where everything is ok. And I had situations where the singing has the loop enabled, when I played the singing after the patter? But I could not reconstruct it, so this does not help so much ;-/. When I have time again, I will test a little more.
